frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/__tests__/PipelineRunDetails.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunDetails } from '../PipelineRunDetails';
+import mockPipelineRun from '../__mocks__/pipeline-run-mock';
+import { PipelineModel, PipelineResourceModel } from '../../../../models';
+import PipelineRunVisualization from '../PipelineRunVisualization';
+
+describe('PipelineRun details page', () => {
+  let wrapper: ShallowWrapper<any>;
```

Comment:
instead of any can we infer actual props and use 

```
type PipelineRunDetailsProps = React.ComponentProps<typeof PipelineRunDetails>;
let wrapper: ShallowWrapper<PipelineRunDetailsProps>;
```